pixz/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Maxim Moseychuk ("fshp") <franchesko.salias.hudro.pedros@gmail.com>
+
+pkgname=pixz
+pkgver=1.0.7
+pkgrel=2
```

Comment:
since this is still a "new" package in the eyes of the repo, I don't think pkgrel should be 2